api/s2n.h

```diff
@@ -494,6 +494,18 @@ extern int s2n_config_set_initial_ticket_count(struct s2n_config *config, uint8_
 S2N_API
 extern int s2n_connection_add_new_tickets_to_send(struct s2n_connection *conn, uint8_t num);
 
+/**
+ * Returns the number of session tickets issued by the server.
```

Comment:
Is this function supposed to work for both server and client connections? If it doesn't, you might want to mention that more explicitly.